### ISSUE TITLE
pgyer set `ipa` path invalid

### ISSUE THREAD

**Issue by Neojoke** [shishirui/fastlane-plugin-pgyer#1]
if gym set `output_directory` path, like "~/Desktop/jenkins_output/xxxx.ipa", pgyer couldn‘t find this path.

**Comment by shishirui** [comment]
After testing, I found that the problem you said did not exist.

I tested by following steps:

Step 1: edit `fastlane/Fastfile`, add `output_directory` to gym:

```
   Lane: beta do
     Gym (use_legacy_build_api: true, output_directory: "/ tmp")
     Pgyer (api_key: "xxxx", user_key: "xxxx")
   End
```

Step 2: Run `bundle exec fastlane beta`:

```
[16:10:40]: Successfully exported and compressed dSYM file
[16:10:40]: Successfully exported and signed the ipa file:
[16:10:40]: /tmp/monitor.ipa
[16:10:40]: -------------------
[16:10:40]: --- Step: pgyer ---
[16:10:40]: -------------------
[16:10:40]: The pgyer plugin is working!
[16:10:40]: build_file: /tmp/monitor.ipa
[16:10:40]: Start upload /tmp/monitor.ipa to pgyer...
[16:11:10]: Upload success. Visit this URL to see: https://www.pgyer.com/aitA
```

As you can see, fastlane automatically finds `/tmp/monitor.ipa` and uploads successfully.

**Comment by Neojoke** [comment]
```
  desc "test builder dev"
  lane :dev do
    gym(scheme: "HBFundTransaction-AdHoc-dev",
        workspace: "HBFundTransaction.xcworkspace",
        output_name: "投客理财Dev") 
    pgyer(api_key: "10ca868ce4702b0a807cd2688a3ff5d4",
        user_key: "fedea72aa2888ce5562ffa29ea2eacce")
    upload_app_to_bugly(file_path: IPA_OUTPUT_PATH,
        app_key:"76861917-fb91-4dca-a588-2af97b001532",
        app_id:"844a24aa62",
        title:"投客理财iOS")
  end
```
This's script of Fastfile,but still failure:
```
[13:42:35]: Do not let descriptions end with a '.', since it's used for user inputs as well

+------+-------------------------------------+-------------+
|                     fastlane summary                     |
+------+-------------------------------------+-------------+
| Step | Action                              | Time (in s) |
+------+-------------------------------------+-------------+
| 1    | Verifying required fastlane version | 0           |
| 2    | default_platform                    | 0           |
| 3    | cocoapods                           | 2           |
| 4    | gym                                 | 43          |
| 💥   | pgyer                               | 0           |
+------+-------------------------------------+-------------+

[13:42:35]: fastlane finished with errors

[!] Do not let descriptions end with a '.', since it's used for user inputs as well
```
sorry,I got confused, spent lot of time for that, hope~

**Comment by shishirui** [comment]
I have released a new version 0.2.1 to fix this bug. Please upgrade you fastlane-plugin-pgyer to 0.2.1.
